broker/sqlstore_broker.go

```diff
@@ -201,7 +204,8 @@ func (b *sqlStoreBroker) processBlock(ctx context.Context, dbContext context.Con
 			return nil, ctx.Err()
 		case err = <-errCh:
 			return nil, err
-
+		case <-slowTimeUpdateTicker.C:
+			b.log.Warningf("slow time update detected, block height: %d, total block processing time: %v", block.Height, blockTimer.duration)
```

Comment:
Might be a handy feature to tell us the time it eventually took? Though we could look in metrics I guess :)

Reply:
I've added to the log line the check interval as you can then infer the time between time updates to a check interval (2s) accurracy if you know this given it will log every check interval,  keeps the metrics code to a minimum.